db/migrate/20160907123728_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table "comments", force: :cascade do |t|
+      t.integer  "incident_id", limit: 4,   null: false
+      t.integer  "user_id",     limit: 4,   null: false
+      t.string   "comment",     limit: 255, null: false
```

Comment:
I think `comment` can be longer then 255. How about changing this to text column?